src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/FindTokenIntegrationTest.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Razor.Language.IntegrationTests;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Language.Test;
+
+public class FindTokenIntegrationTest : IntegrationTestBase
+{
+    [Fact, WorkItem("https://github.com/dotnet/razor/issues/9177")]
+    public void EmptyDirective()
```

Comment:
I can move this to an existing test file if we want. I didn't see any that were a _great_ fit, but it might be fine under `BasicIntegrationTest`.